src/components/NavigationBar/NavigationBar.js

```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+import AppBar from 'material-ui/AppBar';
+import FlatButton from 'material-ui/FlatButton';
+
+class NavigationBar extends Component {
+    
+    constructor() {
```

Comment:
we dont actually need a constructor if its not being used

Reply:
Old habits or whatever. I'll nix it.

Reply:
In: e34b656